web/src/pm_list_data.ts

```diff
@@ -84,8 +84,9 @@ export function get_conversations(search_string = ""): DisplayObject[] {
             unread.num_unread_mentions_for_user_ids_strings(user_ids_string) > 0;
         const is_group = user_ids_string.includes(",");
         const is_active = user_ids_string === active_user_ids_string;
-        const is_deactivated = !people.is_active_user_for_popover(
-            Number.parseInt(user_ids_string, 10) || 0,
+        const user_ids = people.user_ids_string_to_ids_array(user_ids_string);
```

Comment:
This is now called twice (see where `users` is defined above), so we should probably just get the array once and then use that in both places.